### PR TITLE
Modernization

### PR DESCRIPTION
**Pull request by scheul93**
behold the 50+ unit tests
plus support for NPM and Travis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,41 +17,41 @@ A modular, promise-based form validation plugin, free of third-party dependencie
...
-$ jspm install github:DEGJS/formValidation
+$ npm install @degjs/form-validation
 ```
 
-formValidation rule modules that are hosted on GitHub or NPM can similarly be installed with JSPM:
+formValidation rule modules that are hosted on GitHub or NPM can similarly be installed:
 
 ```
-$ jspm install github:DEGJS/formValidation-required
+$ npm install @degjs/form-validation-required
```
Comment (aladage) [inline README.md]: Are you planning to migrate this soon, so it works? :)

File: tests/formValidation.spec.js [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+import state from '../src/utils/state';
+import {emptyElements} from '@degjs/dom-utils';
+
+describe('formValidation', () => {
+    let formEl;
+    let stateInst;
+
+    beforeEach(() => {
+        formEl = document.createElement('form');
+        const fieldEl = document.createElement('div');
```
Comment (aladage) [inline tests/formValidation.spec.js]: Would there be value in appending and testing more than one field, possibly with different rules applied to each?

File: tests/formValidation.spec.js [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+            formValidation(formEl, {});
+            expect(formUtils.checkFormIntegrity).toHaveBeenCalled();
+        });
+
+        it('should disable browser validation', () => {
+            formValidation(formEl, {});
+            expect(formUtils.disableBrowserValidation).toHaveBeenCalled();
+        });
+
+        describe('registerFields', () => {
```
Comment (aladage) [inline tests/formValidation.spec.js]: Do we need to explicitly test the exposed addFields() method, or do you feel like it's covered by the internal call to registerFields()?

Comment (scheul93) [reply tests/formValidation.spec.js]: Since I have a test suite dedicated to registerFields, I feel that is enough to cover addFields

**Comment by aladage** [comment]
Looks great!

**Pull request merged**